x-pack/plugins/task_manager/server/task_type_dictionary.ts

```diff
@@ -8,6 +8,14 @@
 import { TaskDefinition, taskDefinitionSchema, TaskRunCreatorFunction } from './task';
 import { Logger } from '../../../../src/core/server';
 
+/**
+ * Types that are no longer registered and will be marked as unregistered
+ */
+export const REMOVED_TYPES: string[] = [
+  // for testing
+  'sampleTaskRemovedType',
```

Comment:
Once this PR is merged will we not actually have any type in here?
It raises the question of whether we need the unusedTypes feature at all, no? 🤔